tests/test_analysis/test_count_in_voxel.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+from pytraj import *
+import pytraj as pt
+from utils import fn
+from pytraj.testing import aa_eq
+import numpy as np
+
+
+class Test(unittest.TestCase):
```

Comment:
Since you're making several PRs, so it would be better to follow the "modern" way to write test, by using pytest's style.

- The class is not needed
- Change the name of `test_0` to some thing meaningful to you.

Reply:
How do I trigger testing locally? Right now I'm making changes directly to the package that is installed, then running the individual test scripts I'm writing. When I remove the class and `if __name__ == __main__` clause and run the script with just the function definitions, the tests themselves do not get executed.